doc/pairs.md

```diff
@@ -207,3 +207,65 @@ tuples
     - 51
 ...
 ```
+## ``fields`` parameter
+
+With ``fields`` parameter you can choose the value of which fields you would like to get.
+Result contains fields specified by ``fields`` parameter, primary key and scan key. We need primary and scan keys in result to support pagination.
```

Comment:
Result contains only fields specified by ``fields`` parameter, but scan key and primary key values are merged to the result fields to support pagination (any tuple from result can be simply passed to `after` option).
Using `fields` parameters allows to reduce amount of data transferred from storage.